CHANGELOG.md

```diff
@@ -9,6 +9,41 @@ See the full list of issues at [JIRA](https://issues.jenkins-ci.org/issues/?filt
 
 Unreleased
 ----------
+* Check on ContainerWaitingState if the Docker image can't be found, prints message to build console and cancels and removes job from build queue. [#497](https://github.com/jenkinsci/kubernetes-plugin/pull/497) [JENKINS-53790](https://issues.jenkins-ci.org/browse/JENKINS-53790)
+
+1.16.5 and newer
+------
+
+No longer tracked in this file. See [GitHub releases](https://github.com/jenkinsci/kubernetes-plugin/releases) instead.
+
+1.16.4
+------
+* Add http connections count [#528](https://github.com/jenkinsci/kubernetes-plugin/pull/528)
+* Hide deprecated credential type [#529](https://github.com/jenkinsci/kubernetes-plugin/pull/529) [JENKINS-49717](https://issues.jenkins-ci.org/browse/JENKINS-49717)
+* Allow option `showRawYaml` in Jenkinsfiles [#519](https://github.com/jenkinsci/kubernetes-plugin/pull/519)
+
+1.16.3
+------
+* **testing** Fixed test flakes [#496](https://github.com/jenkinsci/kubernetes-plugin/pull/496) [JENKINS-46076](https://issues.jenkins-ci.org/browse/JENKINS-46076)
```

Comment:
Seems you misresolved at least one merge conflict, and there are still several outstanding merge conflicts. If you are not sure how to merge with upstream safely, better to trash the existing commits and force-push a fresh diff. (It is a good idea to check your own PR’s diff tab to verify that you are submitting what you think you are submitting.)